README.mkdn

```diff
@@ -52,8 +52,9 @@ $ repo sync
 # Run to prepare our devices list
 $ . build/envsetup.sh
 # ... now run
-$ brunch "your device codename"
+$ brunch "your device codename" user
```

Comment:
our intention is run brunch with normal parameters, that means userdebug
https://github.com/crdroidandroid/android_vendor_crdroid/blob/13.0/build/envsetup.sh#L83

Reply:
Ok, but that's broken on lemonadep, see https://github.com/crdroidandroid/android/issues/119. The commands given in this repo don't work for building lemonadep. Is there a place for device specific documentation that this could be added to?

Reply:
Now please stop this mambo jambo
https://github.com/crdroidandroid/android/commit/52b9122fb116c406e7d1c018bac293398c72a26d

Reply:
That's very hurtful. I won't interact with you unnecessarily anymore.